src/Morphir/Visual/ViewPatternMatch.elm

```diff
@@ -0,0 +1,138 @@
+--        , { header = text "d", width = fill, view = \example -> text example.d }
+--        , { header = text "e", width = fill, view = \example -> text example.e }
+--        , { header = text "Result", width = fill, view = \example -> text example.result }
+--        ]
+--    }
+--decisionTable : DecisionTable
+--decisionTable =
+--    { decomposeInput = getHeaders param
+--    , rules = []
+--    }
+--
+--
+--getHeaders : Value ta va -> List TypedValue
+--getHeaders =
+--    []
+
+
+type alias Example =
+    { a : String
+    , b : String
+    , c : String
+    , d : String
+    , e : String
+    , result : String
+    }
+
+
+examples : List Example
+examples =
+    [ { a = "foo", b = "bar", c = "any", d = "any", e = "any", result = "1" }
+    , { a = "any", b = "any", c = "bar", d = "any", e = "any", result = "2" }
+    , { a = "any", b = "any", c = "any", d = "any", e = "any", result = "3" }
+    ]
+
+
+
+--flattenTree : Value ta va -> List ( Pattern pa, Value ta va ) -> List (Column record msg)
+--flattenTree param patterns =
+--    patterns |> List.concatMap (extractNestedPattern param)
+--
+--
+--extractNestedPattern : Value ta va -> ( Pattern pa, Value ta va ) -> List (Column record msg)
+--extractNestedPattern param pattern =
+--    case pattern of
+--        ( _, Value.PatternMatch pa nestedParam nestedPattern ) ->
+--            flattenTree nestedParam nestedPattern
+--
+--        _ ->
+--            generateColumn pattern param
+
+
+generateColumn : List ( Pattern pa, Value ta va ) -> Value ta va -> List (Column record msg)
+generateColumn patterns param =
+    case param of
+        Value.Variable ta name ->
+            [ Column (text (nameToText name)) fill (\_ -> text "foo") ]
+
+        Value.Tuple va elems ->
+            elems |> List.concatMap (generateColumn patterns)
+
+        _ ->
+            []
+
+
+generateColumns : Value ta va -> List ( Pattern pa, Value ta va ) -> List (Column record msg)
+generateColumns param patterns =
+    concat [ generateColumn patterns param, [ Column (text "Result") fill (\_ -> text "foo") ] ]
+
+
+
+--getType : Value ta va -> ta
+--getType (Value ta va) =
+--    ta
+--
+--
+
+
+getDecomposedInput : Value ta va -> TypedValue
+getDecomposedInput param =
+    case param of
+        Value.Variable ta name ->
+            [ Value.Variable ta name ]
+
+        --todo: extract nested parameters for nested pattern matches
+        Value.Tuple ta list ->
+            [ param ]
+
+        _ ->
+            []
+
+
+getRules : List ( Pattern pa, TypedValue ) -> List (List ( Match, TypedValue ))
+getRules patterns =
+    []
+
+
+
+--toDecisionTable : Value ta va -> List (Pattern pa, Value ta va ) -> DecisionTable
+--toDecisionTable param patterns =
+--case param of
+--    Value.Variable ta name ->
+--        [ Column (text (nameToText name)) fill (\example -> text "foo") ]
+--
+--    Value.Tuple va elems ->
+--        elems |> List.concatMap (generateColumns patterns)
+--
+--    _ ->
+--        []
```

Comment:
This seems to be showing a hard-coded example instead of what's in the IR. It's fine to only show the column headers in the first round but we shouldn't show something that's not in the IR. 

As a general rule you shouldn't push any code that's commented out.